packages/steppp-react/vite.config.js

```diff
@@ -1,27 +1,8 @@
 import { defineConfig } from "vite";
-import path from "path";
 
 export default defineConfig({
   test: {
     environment: "jsdom",
     setupFiles: "../../setupTests.js",
   },
-  build: {
```

Comment:
No longer needed since Vite will only be responsible for local development and running tests.